README.md

```diff
@@ -23,6 +31,33 @@ cd ios
 pod install
 ```
 
+## API
+
+| Prop name      | Type     | Description                                                                                                       |
+|----------------|----------|-------------------------------------------------------------------------------------------------------------------|
+| items          | array    | An array of objects, each one having the 'icon', 'title' and 'action' keys. Each object represents a menu option. |
+| reportIssue    | function | A callback function invoked when the 'Report a problem' option was selected.                                      |
+| email          | string   | Used to receive the device information after the 'Report a problem' option was selected.                          |
+| reportIcon     | element  | Replaces the default report icon.                                                                                 |
+| dismissIcon    | element  | Replaces the default dismiss icon.                                                                                |
+| containerStyle | object   | Replaces the default container styles.                                                                            |
+| listItemStyle  | object   | Replaces the default style of each item in the list.                                                              |
+
+## Examples
+
+### Example App
+
+Our example app includes all the basic use cases of our component.
```

Comment:
What do you think about this rewording "The example app includes all the basic use cases of the `Shkr` component."?